Riot/Modules/Application/LegacyAppDelegate.h

```diff
@@ -286,4 +286,12 @@ UINavigationControllerDelegate
 - (void)legacyAppDelegate:(LegacyAppDelegate*)legacyAppDelegate wantsToPopToHomeViewControllerAnimated:(BOOL)animated completion:(void (^)(void))completion;
 - (void)legacyAppDelegateRestoreEmptyDetailsViewController:(LegacyAppDelegate*)legacyAppDelegate;
 
+- (void)legacyAppDelegate:(LegacyAppDelegate*)legacyAppDelegate didAddMatrixSession:(MXSession*)session;
```

Comment:
Those methods will be removed later, right?
I asked to be sure to understand.

Reply:
Yes all these delegate methods are here to help transitioning to the `AppCoordinator`, `LegacyAppDelegate` should disappear one day.